sharing/process/index.md

```diff
@@ -0,0 +1,79 @@
+
+If you're new to all this, please don't hesitate to {% include contact.html subject="SGID Data Additions" contact=site.data.contacts.agrc text='reach out to ' hide-punctuation=true %}. We're happy to help guide you through the process.
+
+### SGID Index
+{: .text-left}
+
+It's quite simple to list [qualifying]({% link about/policy/sgid/index.md %}#sgid-index-qualifications) data in the SGID Index. After you submit an issue in Porter using the `Add data to the SGID Index template`, we will review your submission against the qualifications and reach out with any questions. Once we've worked together to resolve any questions or problems, we'll add your dataset to the SGID Index and close the issue.
+
+### Open Data
+{: .text-left}
+
+Adding data to [opendata.gis.utah.gov](https://opendata.gis.utah.gov) requires a few additional steps to navigate the ArcGIS Online sharing mechanics. This time you'll use the `Add data to Open Data` issue template. After the issue is opened, you'll need to request access to the [Open Data Staging Group](https://utah.maps.arcgis.com/home/group.html?id=3d3bd0d238f24f45b2c4b84f1015a317) in ArcGIS Online. This is a staging group that allows us to review the dataset before sharing it through Open Data.
+
+Once we've verified the dataset meets the qualifications, you can share it with the appropriate Utah SGID ArcGIS Online group. All items in these groups are automatically added to Open Data. If this is your first time sharing a data with the particular group, you'll need to request access to the group.
+
+Datasets that languish in the staging group, or that are added to a Utah SGID group without approval, may be removed according to our [policies]({% link about/policy/sgid/index.md %}#reasons-for-removal).
+
+It is critical to provide complete metadata so that Open Data stays clean and easy to use. We've tracked down the sources of the information shown in the Open Data list view to help you understand where the information comes from:
+
+![An annotated Open Data item]({% link images/od_card_anno.png %}){: .flex .flex--center loading="lazy" }
+
+##### A Note about Open Data "Sources"
+
+> You should only have one "Source" for all the datasets coming from your organization. Esri explains where this value comes from [in this FAQ](https://doc.arcgis.com/en/hub/get-started/frequently-asked-questions.htm#GUID-9843B713-46D2-4938-A961-EC0CD81AE410). Basically, it first searches the ArcGIS Online item's metadata for an organization name. If it doesn't find anything there, it falls back to your ArcGIS organization name.
+>
+> To customize the source, we suggest setting it in the `Resource > Citation > Contact > Organization Name` metadata field. Otherwise, leave the metadata contact fields empty so it uses up your ArcGIS Online organization name. We use this second way to show "Utah Automated Geographic Reference Center (AGRC)" as the source for all our datasets.
```

Comment:
```suggestion
> To customize the source, we suggest setting it in the `Resource > Citation > Contact > Organization Name` metadata field. Otherwise, leave the metadata contact fields empty so it uses your ArcGIS Online organization name. We use this second way to show "Utah Automated Geographic Reference Center (AGRC)" as the source for all our datasets.
```